steps/apply.ts

```diff
@@ -513,7 +513,11 @@ export const withdrawAnApplicationAfterSubmission = async (page: Page, applicati
   await expect(page.getByRole('alert', { name: 'Success' })).toContainText('Success')
 }
 
-export const recordAnAppealOnApplication = async (page: Page, applicationId: string) => {
+export const recordAnAppealOnApplication = async (
+  page: Page,
+  applicationId: string,
+  decision: 'Upheld' | 'Rejected',
```

Comment:
Looks like `'Upheld' | 'Rejected'` is used in a few places now - is there a suitable place to abstract this into a named type?